FraihmworkHealthAndIntegrityApi.yaml

```diff
@@ -317,24 +333,25 @@ paths:
       security:
       - SystemWrite:
```

Comment:
Looks like we changed the system endpoint to component, left the system as deprecated, but also left the system security scopes as-is for use with the component endpoint. Should we add a component security scope (double maintenance duty, I know) so we can back out the deprecated API content without having to add a component security scope later? The alternative is, once the system endpoint gets chopped, we have a security scope that has a vestigial name.

Reply:
Updated the scope names